### PR TITLE
[zh] Update securing-a-cluster.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 1]
```diff
@@ -505,8 +505,8 @@ are not encrypted or an attacker gains read access to etcd.
 并考虑在可能的情况下使用全盘加密。
 
 Kubernetes 支持[静态数据加密](/zh-cn/docs/tasks/administer-cluster/encrypt-data/)。
-该功能在 1.7 版本引入，并在 1.13 版本成为 Beta。
-它会加密 etcd 里面的 `Secret` 资源，以防止某一方通过查看 etcd 的备份文件查看到这些
+该功能在 1.7 版本引入，在 1.13 版本成为 Beta, 在 1.25 版本成为 Alpha。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/securing-a-cluster.md:508]:
```suggestion
该功能在 1.7 版本引入，在 1.13 版本成为 V1 Beta, 在 1.25 版本成为 V2 Alpha。
```

**Comment by gaogao101** [comment]
@windsonsea  Thank you for your help. I will finish revising it as soon as possible

File: content/zh-cn/docs/tasks/administer-cluster/securing-a-cluster.md (hunk 1 above)
Comment (mengjiao-liu) [reply content/zh-cn/docs/tasks/administer-cluster/securing-a-cluster.md]: 为什么 `v1` 的字母 `V` 要大写？ 一般不会将其大写吧。@windsonsea

**Comment by gaogao101** [comment]
@mengjiao-liu The modification has been completed. Thank you for help。

File: content/zh-cn/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 2]
```diff
@@ -505,8 +505,8 @@ are not encrypted or an attacker gains read access to etcd.
 并考虑在可能的情况下使用全盘加密。
 
 Kubernetes 支持[静态数据加密](/zh-cn/docs/tasks/administer-cluster/encrypt-data/)。
-该功能在 1.7 版本引入，并在 1.13 版本成为 Beta。
-它会加密 etcd 里面的 `Secret` 资源，以防止某一方通过查看 etcd 的备份文件查看到这些
+该功能在 1.7 版本引入，在 1.13 版本成为 v1 Beta, 在 1.25 版本成为 v2 Alpha。
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/administer-cluster/securing-a-cluster.md:508]:
```suggestion
该功能在 1.7 版引入，在 1.13 版成为 v1 Beta，在 1.25 版成为 v2 Alpha。
```

**Pull request merged**